recipes/tachyon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.99b6" %}
+{% set name = "tachyon" %}
+{% set md5 = "9e5ed4045c472e01e57480d7c7eee849" %}
```

Comment:
Since the home page is down, I've used fedora's link which has the md5 sum in the url. I'll use sha256